services/121-service/src/shared/enum/registration-preferred-language.enum.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * The language a registration/PA prefers to read. Used for messages.
+ * Source: https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes
+ * This is "Set 1" which are all 2 letter codes. The list does change sometimes.
+ */
+export enum RegistrationPreferredLanguage {
+  aa = 'aa',
+  ab = 'ab',
+  ae = 'ae',
+  af = 'af',
+  ak = 'ak',
+  am = 'am',
```

Comment:
why do we include all languages here, and not in the ui-language? Is the difference that the user could theoretically customize to use any language for messages?

Reply:
We want the 121 Platform to communicate with PAs in more languages than the languages we offer the platform in. This is already the case: Ukranian is used as a RegistrationPreferredLanguage but we don't have it as a UI language.